exp/anbox-cloud.md

```diff
@@ -31,9 +31,7 @@ See the following table for a comparison of features for the different variants:
 
 *\* When purchasing the Anbox Cloud Appliance through the AWS Marketplace, the Ubuntu Pro subscription does not include vendor support.*
 
-[note type="information" status="Tip"]
-We recommend to always start with the Anbox Cloud Appliance. You can expand to a full Anbox Cloud installation later.
-[/note]
+Always start with the Anbox Cloud Appliance. You can expand to a full Anbox Cloud installation later.
```

Comment:
Can we go from an established appliance install to a regular deployment without losing anything and needing to redeploy all?

Reply:
No, that's impossible. 
However the above sentence indicates a bit command sense since it's up end user to choose the deployment type. I feel much better if we keep the original one under a tip note as a suggesetion.

Reply:
I have reworded it without adding the "Tip". Does this read better? What do you think @adglkh?

Reply:
That looks good to me. 
Thanks